src/fixture/fixture.test.ts

```diff
@@ -30,20 +31,40 @@ test('creates a fixture', () => {
 	expect(result.totalVisits).toBeTypeOf('number');
 });
 
+test('creates a fixture using createFixture', () => {
+	const result = createFixture(PersonSchema);
+	expect(result.name).toBeTypeOf('string');
+	expect(result.birthday).toBeInstanceOf(Date);
+	expect(result.address).toBeTypeOf('object');
+	expect(result.address.street).toBeTypeOf('string');
+	expect(result.address.city).toBeTypeOf('string');
+	expect(result.address.state).toBeTypeOf('string');
+	expect(result.pets).toBeInstanceOf(Array);
+	expect(result.pets[0]).toBeTypeOf('object');
+	expect(result.pets[0]?.name).toBeTypeOf('string');
+	expect(result.pets[0]?.breed).toBeTypeOf('string');
+	expect(result.totalVisits).toBeTypeOf('number');
+});
+
 test(`priotizes generators via extend`, () => {
-	const FooNumberGenerator = Generator({
-		schema: ZodNumber,
-		filter: ({ context }) => context?.path?.includes('foo'),
-		output: () => 4,
+	const nameGenerator = Generator({
+		schema: ZodString,
+		filter: ({ context }) => context?.path?.includes('name'),
```

Comment:
I don't think this needs to be `context?.path?.includes`. It's old code and can probably be updated to `context.path.includes`.

Reply:
You're correct. Thanks